### DISCUSSION TITLE
屏幕亮度调节不起作用

### DISCUSSION THREAD

**Discussion by Skilling-Kiss** [shuax/MouseInc.Settings#202] [Q&A]
我记得老版本的还能用，说什么升级屏幕亮度调节代码之后就不能用了

**Comment by derQiQ** [comment]
主用亮度调节的话，就定死2.11.3版本吧，其实看日志也没必要用新版因为没啥新增功能、bug修补，之后的版本作者更改亮度调节的调用方式，加减效果也比之前生硬（之前版本亮暗顺滑）。
